### PR TITLE
docs: add progressive link && adjust en docs directory

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: docs/source/en/intro/quickstart.md [hunk 1]
```diff
@@ -408,6 +408,7 @@ with the above short sections.
 We recommend that developers continue reading other documents:
 - Egg provides a powerful mechanism for extending features. See [Plug-ins](../advanced/plugin.md).
 - As large teams need to follow certain constraints and conventions, in egg, we recommend further creating an upper-level framework that fits your team. See [Frameworks](../advanced/framework.md).
+- Egg is a progressive framework, reuse of code is painless. See [Progressive](../tutorials/progressive.md).
```
Comment (atian25) [inline docs/source/en/intro/quickstart.md]: @mattma  SOS

Comment (mattma) [inline docs/source/en/intro/quickstart.md]: @atian25 sure. i am here

Comment (atian25) [inline docs/source/en/intro/quickstart.md]: ^_^ review

File: docs/source/en/intro/quickstart.md [hunk 2]
```diff
@@ -408,6 +408,7 @@ with the above short sections.
 We recommend that developers continue reading other documents:
```
Comment (mattma) [inline docs/source/en/intro/quickstart.md]: Where to go from here? Browse our documentation to better understand the framework.

File: docs/source/en/intro/quickstart.md [hunk 3]
```diff
@@ -408,6 +408,7 @@ with the above short sections.
...
 - Egg provides a powerful mechanism for extending features. See [Plug-ins](../advanced/plugin.md).
 - As large teams need to follow certain constraints and conventions, in egg, we recommend further creating an upper-level framework that fits your team. See [Frameworks](../advanced/framework.md).
```
Comment (mattma) [inline docs/source/en/intro/quickstart.md]: Egg framework allows small or large teams to work together as fast as possible under the well-documented conventions and coding best practices. In addition, the teams can build up logics on top of the framework to better suited their special needs. See more on [Frameworks](../advanced/framework.md).

File: docs/source/en/intro/quickstart.md (hunk 1 above)
Comment (mattma) [inline docs/source/en/intro/quickstart.md]: Egg framework enables developers to write painless unit testing with many plugins and community-powered toolings. The team should give it a try by using Egg unit testing without worrying about setting up the testing tooling but writing the testing logics.

Comment (atian25) [inline docs/source/en/intro/quickstart.md]: this seams to replace the next line?

Comment (atian25) [inline docs/source/en/intro/quickstart.md]: any suggest for `- Egg is a progressive framework, reuse of code is painless. See [Progressive](../tutorials/progressive.md).` ?

Comment (atian25) [inline docs/source/en/intro/quickstart.md]: cool~ 3x

Comment (mattma) [inline docs/source/en/intro/quickstart.md]: By reading at the chinese version, it is only one sentence. I think it should work on both and take it in pieces if you can.

Egg framework provides code reusabilities and modularities. See details at [Progressive](../tutorials/progressive.md).

Comment (atian25) [inline docs/source/en/intro/quickstart.md]: ok. 先重新提交了. 后面英文版翻译的时候, 你们有空再全流程 review 下.

**Comment by atian25** [comment]
我发个文档

**Pull request merged**